css/modals/device-selection/_device-selection.scss

```diff
@@ -0,0 +1,74 @@
+.device-selection {
+    color: $feedbackInputTextColor;
+    display: flex;
+
+    .device-selectors {
+        > div {
+            margin-bottom: 10px;
+        }
+
+        > div:last-child {
+            margin-bottom: 5px;
+        }
+    }
+
+    .device-selection-column {
+        padding: 10px;
+        width: 50%;
+    }
+
+    .device-selection-video-container {
+        background: black;
+        height: 148px;
+        margin-bottom: 5px;
+
+        .video-input-preview {
+            position: relative;
+
+            .video-input-preview-muted {
+                color: $participantNameColor;
+                display: none;
+                left: 0;
+                position: absolute;
+                right: 0;
+                text-align: center;
+                top: 50%;
+            }
+
+            &.video-muted .video-input-preview-muted {
+                display: block;
+            }
+
+            .video-input-preview-display {
+                height: 100%;
+                overflow: hidden;
+                width: 100%;
+            }
+        }
+    }
+
+    .audio-output-preview {
+        text-align: right;
+
+        a {
+            cursor: pointer;
+            text-decoration: none;
+        }
+    }
+
+    .audio-input-preview {
+        background: #f4f5f7;
+        border-radius: 5px;
+        height: 7px;
+
+        .audio-input-preview-level {
+            background: #0052cc;
+            border-radius: 5px;
+            height: 100%;
+            -webkit-transition: width .1s ease-in-out;
```

Comment:
Non actionable note: I wonder if there is an equivalent to gulp-autoprefixer which matches our workflow so we didn't have to do this.